pkg/kubectl/describe.go

```diff
@@ -1201,7 +1190,55 @@ func (d *HorizontalPodAutoscalerDescriber) Describe(namespace, name string) (str
 	})
 }
 
-func filterNonRunningPods(pods []*api.Pod) []*api.Pod {
+func describeNodeResource(pods []*api.Pod, node *api.Node, out io.Writer) {
+	nonTerminatedPods := filterTerminatedPods(pods)
+	fmt.Fprintf(out, "Non-terminated Pods:\t(%d in total)\n", len(nonTerminatedPods))
+	fmt.Fprint(out, "  Namespace\tName\t\tCPU Requests\tCPU Limits\tMemory Requests\tMemory Limits\n")
+	fmt.Fprint(out, "  ─────────\t────\t\t────────────\t──────────\t───────────────\t─────────────\n")
+	totalMilliCPUReq := int64(0)
+	totalMemoryReq := int64(0)
+	fractionPodCPUReq := float64(0)
+	fractionPodMemoryReq := float64(0)
+	fractionTotalCPUReq := float64(0)
+	fractionTotalMemoryReq := float64(0)
+	totalMilliCPULimit := int64(0)
+	totalMemoryLimit := int64(0)
+	fractionPodCPULimit := float64(0)
+	fractionPodMemoryLimit := float64(0)
+	for _, pod := range pods {
+		podTotalMilliCPUReq := int64(0)
+		podTotalMemoryReq := int64(0)
+		podTotalMilliCPULimit := int64(0)
+		podTotalMemoryLimit := int64(0)
+
+		for ix := range pod.Spec.Containers {
+			requests := pod.Spec.Containers[ix].Resources.Requests
+			podTotalMilliCPUReq += requests.Cpu().MilliValue()
+			podTotalMemoryReq += requests.Memory().Value()
+
+			limits := pod.Spec.Containers[ix].Resources.Limits
+			podTotalMilliCPULimit += limits.Cpu().MilliValue()
+			podTotalMemoryLimit += limits.Memory().Value()
+		}
+		totalMilliCPUReq += podTotalMilliCPUReq
+		totalMemoryReq += podTotalMemoryReq
+		totalMilliCPULimit += podTotalMilliCPULimit
+		totalMemoryLimit += podTotalMemoryLimit
+		fractionPodCPUReq = float64(podTotalMilliCPUReq) / float64(node.Status.Capacity.Cpu().MilliValue()) * 100
+		fractionPodMemoryReq = float64(podTotalMemoryReq) / float64(node.Status.Capacity.Memory().Value()) * 100
+		fractionPodCPULimit = float64(podTotalMilliCPULimit) / float64(node.Status.Capacity.Cpu().MilliValue()) * 100
+		fractionPodMemoryLimit = float64(podTotalMemoryLimit) / float64(node.Status.Capacity.Memory().Value()) * 100
+		fmt.Fprintf(out, "  %s\t%s\t\t%dm (%d%%)\t%dKi (%d%%)\t%dm (%d%%)\t%dKi (%d%%)\n", pod.Namespace, pod.Name,
```

Comment:
The units are incorrect here for CPU vs memory.

For example:

```
Non-terminated Pods:		(6 in total)
  Namespace			Name						CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ─────────			────						────────────	──────────	───────────────	─────────────
  default			nginx-6x09m					100m (5%)	0Ki (0%)	0m (0%)		0Ki (0%)
  default			nginx-ejibk					100m (5%)	0Ki (0%)	0m (0%)		0Ki (0%)
  kube-system			kube-dns-v9-efnpt				310m (15%)	310Ki (15%)	178257m (17%)	178257Ki (17%)
  kube-system			kube-ui-v2-gq3ln				100m (5%)	100Ki (5%)	52428m (5%)	52428Ki (5%)
  kube-system			monitoring-heapster-v8-pehu9			100m (5%)	100Ki (5%)	314572m (30%)	314572Ki (30%)
  kube-system			monitoring-influx-grafana-v1-9bldm		200m (10%)	200Ki (10%)	314572m (30%)	314572Ki (30%)
```

Note that you are showing Ki for CPU Limits and and m for Memory Requests.

I would prefer you update this code to build a resource.Quantity object for the resource type, and then just print out the resource.Quanity using the internal string representation library which will output units next to the value.